gym_snake/envs/snakeGameGym.py

```diff
@@ -22,12 +22,15 @@ class SnakeGameGym(SnakeGame):
 	def __init__(self,
```

Comment:
Got carried away on naming stuff, thought some changes might make variables shorter and clearer, but don't worry about it if you think otherwise!

Reply:
I agree with the naming changes and just pushed them 👍